### DISCUSSION TITLE
STUdio

### DISCUSSION THREAD

**Discussion by manuelchavanet** [marian-m12l/studio#148] [Q&A]
Bonjour Mariam, 
J'ai soucis pour afficher la luuni dans la partie gauche de l'écran ,je suis débutant et je ne trouve pas la solution ,j'ai bien réussi à créer un pack d'histoire mais impossible de le transférer.merci d'avance pour vos retours ,et merci pour cet outil super
<img width="1440" alt="Capture d’écran 2020-12-25 à 17 01 28" src="https://user-images.githubusercontent.com/76585639/103138534-e04b3100-46d3-11eb-9c41-59e7d640e195.png">

**Comment by kenavy** [comment]
Bonjour,

J'ai également le même problème.

**Comment by marian-m12l** [comment]
Bonjour,

Le nouveau firmware (2.x) n'est pas encore supporté. Vous pouvez suivre l'évolution sur le ticket #122

**Comment by xbertrand** [comment]
Bonjour, Si vous avez l'ancien firmware, sous windows, c'est peut être simplement le fichier de lancement *.bat qu'il faut modifier pour ajouter la lettre du lecteur où est branchée la Lunii :
:: Configure mount point of the device
set FS_MOUNTPOINT=F:\

**Reply by marian-m12l** [reply]
Le point de montage est bien nécessaire pour le nouveau firmware 2.x, quelle que soit la version du matériel (made in china ou made in france)
